### ISSUE TITLE
Add images to guidelines

### ISSUE THREAD

**Issue by TeresaMcGervey-USDA** [dorisavedikian/NALGuideToSubjectIndexing#9]
Since I cannot access the indexing guidelines book, I will let you know where to put the images in the text. I'll attach both the images (I uploaded these a couple of different ways but none looked right) and the link to where you can find the image so you can download it if that will be easier. Let me know if there is anything that does not make sense or will not work.

In Supplement D, Similar NALT Terms:  **choroid** (not a NALT term, MESH BT **eyes** -> **uvea**) versus **choroid plexus** (https://lod.nal.usda.gov/nalt/25709, BT **nervous system** -> **plexus**) 

Choroid - https://www.cancer.gov/publications/dictionaries/cancer-terms/def/choroid
![choroid](https://user-images.githubusercontent.com/69479996/232475189-e52aca0f-1bf0-4db6-8c96-519c324adf83.jpg)

Choroid plexus - https://www.cancer.gov/publications/dictionaries/cancer-terms/def/choroid-plexus
![choroid plexus](https://user-images.githubusercontent.com/69479996/232475304-c6662843-62f7-40a4-ad8a-0168e797032f.jpg)

**Comment by TaMcG412** [comment]
Edit the images as needed.

**Comment by dorisavedikian** [comment]
Got em. Thanks Teresa. I will let you know when the site is working again. By any chance, do you and the team have a git lab account?

**Comment by TaMcG412** [comment]
I'm not sure. I will check with Jennifer.

**Comment by dorisavedikian** [comment]
> 

Ok the website is back up. check it out and let me know if you like how I changed the format for the article and also added the pictures above. [Site](https://dorisavedikian.github.io/NALGuideToSubjectIndexing/)

**Comment by TaMcG412** [comment]
Looks good! I'll send you another image (probably a map) later to see how that works.

**Comment by TaMcG412** [comment]
I posted annotations for two maps (PDFs below) in Geographic Descriptors section.

[Map - Hawaii.pdf](https://github.com/dorisavedikian/NALGuideToSubjectIndexing/files/11277310/Map.-.Hawaii.pdf)

[Map - Western Virginia.pdf](https://github.com/dorisavedikian/NALGuideToSubjectIndexing/files/11277312/Map.-.Western.Virginia.pdf)

**Comment by dorisavedikian** [comment]
> I posted annotations for two maps (PDFs below) in Geographic Descriptors section.
> 
> [Map - Hawaii.pdf](https://github.com/dorisavedikian/NALGuideToSubjectIndexing/files/11277310/Map.-.Hawaii.pdf)
> 
> [Map - Western Virginia.pdf](https://github.com/dorisavedikian/NALGuideToSubjectIndexing/files/11277312/Map.-.Western.Virginia.pdf)

@TaMcG412 Are you able to convert those to .png's? I don't have Adobe on my computer so I can't change those pdfs to a .png file.

**Comment by dorisavedikian** [comment]
@TaMcG412 I took a screen shot of each pdf and saved the screen shot as a .png image file. The map of west virginia look okay like this but the map of the island of Hawaii looks a little blurry after this. If you can save the image of Hawaii as a .png and send that to me - it might look better.

**Comment by TaMcG412** [comment]
One thing I forgot about most articles is that most publishers include the figures as separate JPEGs (along with the  HTML and PDF files) so I've attached those. There is no text with the JPEGs but that isn't critical for the guidelines. It was included on the PDF images since I cropped those from the full article. 

![Western Virginia - map](https://user-images.githubusercontent.com/69479996/233348645-28729035-fc87-44c2-b602-0b3e68994bef.jpg)

![Hawaii - map](https://user-images.githubusercontent.com/69479996/233348682-ff6a4d24-746a-4d74-985d-99923e9bc926.jpg)
